ui/src/app/workflows/components/workflow-filters/workflow-filters.tsx

```diff
@@ -33,13 +63,25 @@ export class WorkflowFilters extends React.Component<WorkflowFilterProps, {}> {
                         <p className='wf-filters-container__title'>Labels</p>
                         <TagsInput
                             placeholder=''
-                            autocomplete={this.getLabelSuggestions(this.props.workflows)}
+                            autocomplete={this.labelSuggestion}
                             tags={this.props.selectedLabels}
                             onChange={tags => {
                                 this.props.onChange(this.props.namespace, this.props.selectedPhases, tags);
                             }}
                         />
                     </div>
+                    <div className='columns small-3 xlarge-12'>
+                        <p className='wf-filters-container__title'>Workflow Template</p>
```

Comment:
I think we should extract these two dropdowns into a shared component.

Reply:
- added screenshot
- added dropdown share component